egs/zeroth_korean/s5/local/data_prep.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# Copyright  2018  Atlas Guide (Author : Lucas Jo)
+#            2018  Gridspace Inc. (Author: Wonkyum Lee)
+# Apache 2.0
+
+# Modified by Lucas Jo 2017 (Altas Guide)
+
+if [ "$#" -ne 2 ]; then
+  echo "Usage: $0 <src-dir> <dst-dir>"
+  echo "e.g.: $0 ./db/train_data_01 data/train_data_01"
+  exit 1
+fi
+
+src=$1
+dst=$2
+
+# all utterances are FLAC compressed
+if ! which flac >&/dev/null; then
+   echo "Please install 'flac' on ALL worker nodes!"
+   exit 1
+fi
+
+spk_file=$src/../AUDIO_INFO
```

Comment:
I don't very much like how you are going one level up from "$src".  If one-level-up is the real source, you should give that directory.  But make sure that entire directory is downloaded from one place.  If it's multiple downloads then make them 2 separate arguments to this script.

Reply:
Whole data is downloaded in one place. It was trying to refer meta data while preparing data directory. Anyway, I fixed it.